pyth/tests/test_publish.py

```diff
@@ -54,3 +64,68 @@ def get_price_acct():
     assert after['publisher_accounts'][0]['price'] == 150
     assert after['publisher_accounts'][0]['conf'] == 7
     assert after['publisher_accounts'][0]['status'] == 'trading'
+    
+    #constants from oracle.h
+    PROGRAM_VERSION = 2 #TODO: update this
+    COMMAND_UPD_ACCOUNT = 14
+    SYSTEM_PROGRAM = "11111111111111111111111111111111"
+
+    #update version of price accounts to make sure they resize
+    layout = Struct("version" / Int32ul, "command" / Int32sl)
+    data = layout.build(dict(version=PROGRAM_VERSION, command=COMMAND_UPD_ACCOUNT))
+    funding_key = PublicKey(solana_keygen[0])
+    price_key = PublicKey(before["account"])
+    system_key = PublicKey(SYSTEM_PROGRAM)
+    print("program id is", solana_program_deploy)
+    resize_instruction = TransactionInstruction(
+        data=data,
+        keys=[
+            AccountMeta(pubkey=funding_key, is_signer=True, is_writable=True),
+            AccountMeta(pubkey=price_key, is_signer=False, is_writable=True),
+            AccountMeta(pubkey=system_key, is_signer=False, is_writable=False),
+        ],
+        program_id = PublicKey(solana_program_deploy),
+    )
+    print("key pair", type(solana_keygen[1]), solana_keygen[1])
+    txn = Transaction().add(resize_instruction)
+    solana_client = Client("http://localhost:8899")
+    key_file = open(solana_keygen[1])
+    key_data = json.load(key_file)
+    key_file.close()
+
+    sender = Keypair.from_secret_key(key_data)
+    txn.sign(sender)
+    print(solana_client.send_transaction(txn, sender))
+
```

Comment:
You might want to query the account and check the size is what you want it to be